src/background/background.js

```diff
@@ -0,0 +1,41 @@
+import { initDB } from './initdb.js';
+import { fetchAndStoreHistory } from './handlers/fetchAndStoreHistory.js';
```

Comment:
that's a lot of modules and imports.

I would merge all handlers in a single file and then export a single top var that would publish all these functions.